actors/evm/src/interpreter/precompiles/parameter.rs

```diff
@@ -85,6 +93,7 @@ impl<'a> TryFrom<&'a [u8; 32]> for Parameter<u8> {
 
     fn try_from(value: &'a [u8; 32]) -> Result<Self, Self::Error> {
         assert_zero_bytes::<31>(value)?;
+        log::trace!("Read u8: {}", value[31]);
```

Comment:
Unless we disable trace logging _at compile time_ (which we should consider?), this will almost certainly have a performance impact.

Reply:
But regardless, things that look like pure functions should almost never log (especially conversions). This will make it very difficult for the compiler to optimize (e.g., by forcing memory barriers through atomic lookups).